assets/less/page.less

```diff
@@ -6,14 +6,43 @@
 }
 
 .page-title {
-  h1 {
+  border-bottom: 1px solid #f2f2f3;
+  text-align: center;
+  h2 {
     margin-top: 0;
   }
+  h3 {
+    margin-bottom: 0;
+  }
+}
+
+article {
   h2 {
-    margin-bottom: 10px;
+    margin-bottom: 0;
   }
+}
 
-  @media @desktop {
-    width: 50%;
+// For styling blog posts coming from BernieSanders.com official campaign
+.official {
+  h2 {
+    // Add text to denote official source
+    &:before {
+      content: 'Official News';
```

Comment:
Not really - I did it that way to keep the home.php page cleaner, but I think it would actually be better in the markup. PR incoming.